python/ray/autoscaler/_private/aws/config.py

```diff
@@ -719,6 +723,51 @@ def _get_key(key_name, config):
         raise exc
 
 
+def _configure_from_network_interfaces(config):
+    for cfg_key in NODE_KIND_CONFIG_KEYS.values():
```

Comment:
Hmmm, I think we need to run this over `config["available_node_types"]` too right?